### PR TITLE
do not reduce the lookup scope when getting highlights for document elements

### PR DESCRIPTION
**Pull request by vladima**
workaround for #4255.

### CODE REVIEW

File: src/services/services.ts [hunk 1]
```diff
@@ -5247,7 +5247,7 @@ namespace ts {
 
             // Try to get the smallest valid scope that we can limit our search to;
             // otherwise we'll need to search globally (i.e. include each file).
-            let scope = getSymbolScope(symbol);
+            let scope = reduceScopeIfPossible && getSymbolScope(symbol);
```
Comment (mhegazy) [inline src/services/services.ts]: i would just remove the handling for the private property case from getSymbolScope..

**Comment by vladima** [comment]
I still would like to keep it for FindAllRefs scenario

**Comment by mhegazy** [comment]
I would just make sure we have the search location in the output. It is not clear when should u use this flag. Limiting the scope is just an optimization and should not have functional impact.

**Comment by vladima** [comment]
I think I have a better idea for the fix, closing this PR

**Pull request closed without merging**